internal/test/e2e/otelglobal/traces.json

```diff
@@ -65,6 +65,15 @@
               "status": {},
               "traceId": "xxxxx"
             },
+            {
```

Comment:
you can use `make fixture-otelglobal` to tun the test locally.

Reply:
the e2e tests weren't fully working locally like i thought they were. i was able to get things working on my macbook, minus the e2e tests. even on main, i'm seeing the following logs whenever i try to run `make fixture-otelglobal` locally on my macbook. any ideas?

```
{"level":"error","ts":1704932079.780636,"logger":"Instrumentation.Manager","caller":"instrumentation/manager.go:175","msg":"error while loading probes, cleaning up","name":"go.opentelemetry.io/otel/internal/global/client","error":"field UprobeSetAttributes: program uprobe_SetAttributes: load program: argument list too long: regs=100 stack=0 before 136: (6b) *(u16 *)(r10 -8) = r1: regs=100 stack=0 bef (truncated, 1189 line(s) omitted)","stacktrace":"go.opentelemetry.io/auto/internal/pkg/instrumentation.(*Manager).load\n\t/app/internal/pkg/instrumentation/manager.go:175\ngo.opentelemetry.io/auto/internal/pkg/instrumentation.(*Manager).Run\n\t/app/internal/pkg/instrumentation/manager.go:125\ngo.opentelemetry.io/auto.(*Instrumentation).Run\n\t/app/instrumentation.go:152\nmain.main\n\t/app/cli/main.go:108\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:267"}
```

Reply:
is it possible to use CI to troubleshoot the e2e tests? it looks like that's blocked on approval. i'm trying a few other things on my end, but want to see if i have that as a fallback option.

Reply:
nevermind - third try's the charm! i installed a new dev env using wsl on my windows box and was able to get tests working fine locally. CI should pass now.